### ISSUE TITLE
GOAL: Assess 30-40 DAOs

### ISSUE THREAD

**Issue by vincenzospaghetti** [Ledgerback/DGSF#15]

**Comment by linxule** [comment]
@kylefox1 do you have a dedicated Airtable base or equivalent where we can leave comments?

Alternatively, we can leverage the base that I created. But I'm unclear if the original questions are up to date. 
- https://airtable.com/appxwnyzem8EAbGtk/tblHVUkQTReLlfZm2/viwNIMm7TFEQqelHu?blocks=hide
You should all have access to this already.

**Comment by kylefox1** [comment]
> @kylefox1 do you have a dedicated Airtable base or equivalent where we can leave comments?

Yes. Please refer to the *Questionnaire-V09-Draft_Assessment-2023_08_16* table in the *V0.9-DAO_Index-Assessment_Instrument-Scorecard_Toolkit-2024_03_21* base

**Comment by linxule** [comment]
> > @kylefox1 do you have a dedicated Airtable base or equivalent where we can leave comments?
> 
> 
> 
> Yes. Please refer to the *Questionnaire-V09* table in the *DGSF-DAO_Index-Resources-2024_03_07* base on NocoDB.

Can you attach the links in your reply? Many thanks

**Comment by kylefox1** [comment]
> > > @kylefox1 do you have a dedicated Airtable base or equivalent where we can leave comments?
> > 
> > 
> > Yes. Please refer to the _Questionnaire-V09_ table in the _DGSF-DAO_Index-Resources-2024_03_07_ base on NocoDB.
> 
> Can you attach the links in your reply? Many thanks

You can find the base here: https://data.ledgerback.xyz/dashboard/#/base/cea9c5a9-4b76-4478-b9d5-f0f83d941939

**Comment by kylefox1** [comment]
Should I add a form for assessing the question's content validity, based on a 3-point "scale of 'not necessary', 'helpful but not essential', and 'essential'", and a form for a "4-point scale for relevance, clarity, ambiguity, and simplicity (from 1: not relevant, not simple, doubtful, not clear to 4: extremely relevant, extremely simple, meaningful, very clear)"?

Reference:

1. [A Novel Scale for Evaluating Digital Readiness toward Earthquakes: A Comprehensive Validity and Reliability Analysis](https://www.mdpi.com/2071-1050/16/1/252) 

> To further evaluate the scale’s content validity, each item was evaluated during the quantitative phase using the content validity ratio (CVR) and content validity index (CVI). To calculate the CVR, experts were asked to assign each item a score between 1 and 3 on a scale of “not necessary”, “helpful but not essential”, and “essential”, respectively. Each item was acceptable if its CVR exceeded Lawshe’s requirements; otherwise, it was removed. Using Lawshe’s threshold, the crucial CVR value for 9 experts was 0.78 [[39](https://www.mdpi.com/2071-1050/16/1/252#B39-sustainability-16-00252)]. The item content validity index (I-CVI) and scale content validity (S-CVI/Ave) were then examined using a 4-point scale for relevance, clarity, ambiguity, and simplicity (from 1: not relevant, not simple, doubtful, not clear to 4: extremely relevant, extremely simple, meaningful, very clear) [[40](https://www.mdpi.com/2071-1050/16/1/252#B40-sustainability-16-00252),[41](https://www.mdpi.com/2071-1050/16/1/252#B41-sustainability-16-00252),[42](https://www.mdpi.com/2071-1050/16/1/252#B42-sustainability-16-00252)]. An I-CVI of 0.78 or above was regarded as acceptable, whereas values below 0.7 were ruled unsuitable and hence removed, and lastly, values outside this range were corrected [[41](https://www.mdpi.com/2071-1050/16/1/252#B41-sustainability-16-00252),[43](https://www.mdpi.com/2071-1050/16/1/252#B43-sustainability-16-00252),[44](https://www.mdpi.com/2071-1050/16/1/252#B44-sustainability-16-00252)]. Acceptable S-CVI/Ave values as the averages of all I-CVIs were more than or equal to 0.90.

**Comment by linxule** [comment]
Whatever works for you. A framework like this could help me examine the questions more closely. But, I would say feedback on questions would need explanation too, going beyond the simple scale. And most importantly, we would need suggestions on how to improve the questions.

**Comment by kylefox1** [comment]
> Whatever works for you. A framework like this could help me examine the questions more closely. But, I would say feedback on questions would need explanation too, going beyond the simple scale. 

I think adding a field for free form text could suffice then.

**Comment by vincenzospaghetti** [comment]
@kylefox1 are you wanting us to review just these questions:
https://data.ledgerback.xyz/dashboard/#/base/cea9c5a9-4b76-4478-b9d5-f0f83d941939/mbm3gnind0gl9h8

**Comment by kylefox1** [comment]
> @kylefox1 are you wanting us to review just these questions: https://data.ledgerback.xyz/dashboard/#/base/cea9c5a9-4b76-4478-b9d5-f0f83d941939/mbm3gnind0gl9h8

Yes. Please use that table to add your comments on the questions.

**Comment by linxule** [comment]
@kylefox1 can you give us edit access? I don't see an option to add columns or edit text at the moment.

**Comment by kylefox1** [comment]
> @kylefox1 can you give us edit access? I don't see an option to add columns or edit text at the moment.

You should have access. Make sure to login to data.ledgerback.xyz, and then look for the DGSF-DAO_Index-Resources-2024_03_07 base.
